misc/completion/ipfs-completion.bash

```diff
@@ -1,9 +1,18 @@
+_do_comp()
+{
+  if [ -n "$ZSH_VERSION" ]; then
+    complete $@
+  else
+    compopt $@
+  fi
```

Comment:
```suggestion
  if [[ $(type compopt) == *"builtin" ]]; then
    compopt $@
  else
    complete $@
  fi
```

Reply:
perhaps do this by seeing which one is available, rather than an env var

Reply:
I tried looking for something similar but couldn't find anything that worked in both bash and zsh. This suggestion does not work in zsh.